sendsmaily.php

```diff
@@ -250,3 +250,12 @@ function smaily_nojs_subscribe_callback() {
 }
 add_action( 'admin_post_nopriv_smly', 'smaily_nojs_subscribe_callback' );
 add_action( 'admin_post_smly', 'smaily_nojs_subscribe_callback' );
+
+$path = plugin_basename( __FILE__ );
+// display plugin upgrade notification
+add_action("after_plugin_row_{$path}", function( $plugin_file, $plugin_data, $status ) {
```

Comment:
Using `after_plugin_row_[file]` hook does not work well, because the notice will be on separate table row and there is no way user can associate it with this plugin.

I suggest using `plugin_row_meta` filter [hook](https://developer.wordpress.org/reference/hooks/plugin_row_meta/) to just output notice before the plugin meta data.